daemon/container_unix.go

```diff
@@ -948,7 +948,8 @@ func (container *Container) connectToNetwork(idOrName string, updateSettings boo
 
 	ep, err := container.getEndpointInNetwork(n)
 	if err == nil {
-		return fmt.Errorf("container already connected to network %s", idOrName)
```

Comment:
IMO, being unique should not be a responsibility of the network-layer though, the container/graph storage should be responsible for that.

In that particular issue, the error only brought to light that the container/graph storage failed to do so (which was a lucky side effect).

Just my 0.02c, and highly opinionated :smile: